tests/page-objects/common/common.po.js

```diff
@@ -146,17 +168,53 @@ module.exports = {
     }
   },
 
+  goToReportsNative: async (refresh) => {
+    await browser.get(utils.getBaseUrl() + 'reports/');
+    helper.waitElementToPresent(
+      element(
+        by.css('.action-container .general-actions:not(.ng-hide) .fa-plus')
+      )
+    );
+    helper.waitElementToBeClickable(
+      element(
+        by.css('.action-container .general-actions:not(.ng-hide) .fa-plus')
+      )
+    );
+    helper.waitElementToBeVisible(element(by.id('reports-list')));
+    if (refresh) {
+      await browser.refresh();
+    } else {
+      // A trick to trigger a list refresh.
+      // When already on the "reports" page, clicking on the menu item to "go to reports" doesn't, in fact, do anything.
+      await element(by.css('.reset-filter')).click();
+      browser.waitForAngular();
+    }
+  },
+
   goToTasks: () => {
+    utils.deprecated('common.po.goToTasks', 'commmon.po.goToTasksNative');
     browser.get(utils.getBaseUrl() + 'tasks/');
     helper.waitUntilReady(medicLogo);
     helper.waitUntilReady(element(by.id('tasks-list')));
   },
 
+  goToTasksNative: async () => {
+    await browser.get(utils.getBaseUrl() + 'tasks/');
+    await helper.waitUntilReady(medicLogo);
+    await helper.waitUntilReady(element(by.id('tasks-list')));
+  },
+
   isAt: list => {
+    utils.deprecated('common.po.isAt', 'commmon.po.isAtNative');
     helper.waitUntilReady(medicLogo);
     return element(by.id(list)).isPresent();
   },
 
+  isAtNative: async (locator) => {
+    await helper.waitUntilReady(medicLogo);
+    return await element(by.css(locator)).isPresent();
```

Comment:
I would chose to either remove this `isAtNative` method and await it elsewhere or remove the return here.